examples/go/run.sh

```diff
@@ -0,0 +1,18 @@
+set -x
+set -e
+
+# Start artipie.
+docker run --rm -d --name artipie -it -v $(pwd)/artipie.yaml:/etc/artipie.yml -v $(pwd):/var/artipie -p 8080:80 artipie/artipie:latest
+
+# Wait for container to be ready for new connections.
+sleep 5
+
+# Force go client to use Aritpie Go registry.
+export GO111MODULE=on
+export GOPROXY=http://localhost:8080/my-go-proxy
```

Comment:
@Sammers21 I see that Go uses word proxy for custom artifacts manager, but it is a bit misleading in context of Artipie. We have proxy type of adapters and someday going to have proxy for Go as well. I'd suggest to remove proxy word from repo name